test/benchmark/bench_linux_and_aster.sh

```diff
@@ -68,51 +85,27 @@ run_benchmark() {
         -initrd ${BENCHMARK_DIR}/../build/initramfs.cpio.gz \
         -drive if=none,format=raw,id=x0,file=${BENCHMARK_DIR}/../build/ext2.img \
         -device virtio-blk-pci,bus=pcie.0,addr=0x6,drive=x0,serial=vext2,disable-legacy=on,disable-modern=off,queue-size=64,num-queues=1,config-wce=off,request-merging=off,write-cache=off,backend_defaults=off,discard=off,event_idx=off,indirect_desc=off,ioeventfd=off,queue_reset=off \
-        -append 'console=ttyS0 rdinit=/benchmark/benchmark_entrypoint.sh mitigations=off hugepages=0 transparent_hugepage=never' \
+        -append 'console=ttyS0 rdinit=/benchmark/common/bench_runner.sh ${benchmark} linux mitigations=off hugepages=0 transparent_hugepage=never' \
```

Comment:
I am surprised that Linux kernel will take, as the value for the `rdinit` argument, the entire string of `/benchmark/common/bench_runner.sh ${benchmark} linux`, which contains whitespace. Are you sure that it works?

Reply:
I am sure. If we add `echo "Benchmark name: ${BENCH_NAME}";echo "System type: ${SYSTEM}"` to 
 `./benchmark/common/bench_runner.sh`, we can get: 
```log
# Asterinas
Benchmark name: lmbench/pipe_bw
System type: asterinas

# Linux
[    0.879249] Run /benchmark/common/bench_runner.sh as init process
Benchmark name: lmbench/pipe_bw
System type: linux
```